tests/acceptance/features/webUITrashbin/trashbinDelete.feature

```diff
@@ -16,23 +16,23 @@ Feature: files and folders can be deleted from the trashbin
     And the user has browsed to the trashbin page
 
   @smokeTest
-  @skip @yetToImplement
```

Comment:
let's leave the `@yetToImplement` tag if there are things that are not yet implemented in phoenix, or delete those steps if they will not be implemented in the foreseen future